pkg/api/handlers/compat/images.go

```diff
@@ -231,6 +232,12 @@ func CreateImageFromImage(w http.ResponseWriter, r *http.Request) {
 
 	fromImage := mergeNameAndTagOrDigest(query.FromImage, query.Tag)
 
+	_, err := shortnames.Resolve(runtime.SystemContext(), fromImage)
```

Comment:
Can you drop a comment and elaborate why do an early check here?

Future readers may feel inclined to drop the check.